src/pages/event.js

```diff
@@ -254,9 +327,12 @@ function CreateTable({loading, pageCount: pc, columns, data, event}) {
             prepareRow(row)
             return (
               <tr {...row.getRowProps()}>
-                {row.cells.map(cell => {
-                  return <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
-                })}
+                {row.cells.map((cell, idx) => {
```

Comment:
There are several places where you use a list but forgot to use the key prop.

`Each child in a list should have a unique "key" prop`